src/components/Notifier/Notifier.tsx

```diff
@@ -0,0 +1,237 @@
+import {
```

Comment:
Same than Notification.tsx I think is better practices if we cout in many file and scoop responsibility of function by file